autoconfigure/src/main/resources/zipkin-server-kafkastore.yml

```diff
@@ -8,16 +8,19 @@ zipkin:
       compression-type: ${KAFKA_STORE_COMPRESSION_TYPE:NONE}
       retention-scan-frequency: ${KAFKA_STORE_RETENTION_SCAN_FREQUENCY:86400000}
       retention-max-age: ${KAFKA_STORE_RETENTION_MAX_AGE:604800000}
-      trace-inactivity-gap: ${KAFKA_STORE_TRACE_INACTIVITY_GAP:300000}
-      spans-topic: ${KAFKA_STORE_SPANS_TOPIC:zipkin-spans}
+      trace-inactivity-gap: ${KAFKA_STORE_TRACE_INACTIVITY_GAP:60000}
+      spans-topic: ${KAFKA_STORE_SPANS_TOPIC:zipkin-spans-v1}
```

Comment:
what does the v1 mean here?

Reply:
version 1 of kafka storage topics, not related to zipkin api versions at least. I'm just used to version topics :)